### ISSUE TITLE
Possible conflict with Anowave's google tag manager module

### ISSUE THREAD

**Issue by kerlama** [craftyclicks/magento2#18]
Hi,

on magento 2.3.6 with default checkout and anowave's google tag manager module, there is an issue when trying to place the order: Uncaught (in promise) TypeError: Cannot read property 'checkout' of undefined

which is coming from Anowave's place-order.js file:

data.ecommerce.checkout.actionField.step = AEC.Const.CHECKOUT_STEP_ORDER;

This started to happen when we installed latest fetchify module, when its disabled, issue is gone . Wondering if there is a conflict somewhere between these two ?

Thanks

**Comment by Jack-Fetchify** [comment]
Hi

Can we ask you to open a ticket with us directly rather than discuss here on GitHub? Please email support@fetchify.com

When you open the ticket, please provide a store URL or the anowave module itself. A URL is preferable (at least at first).

Thanks
Jack

**Comment by kerlama** [comment]
Hi, as I were short in time ... I had to resolve the conflict myself.

http://prntscr.com/10zqxnp

The problem was in cc_ukpcl_lib.js which have object variable called "data = {}". This variable already exists in anowave's module: https://www.anowave.com/marketplace/magento-2-extensions/magento-2-google-tag-manager-enhanced-ecommerce-ua-tracking/

So I gave that variable unique name. Problem solved.

Ivan
